### DISCUSSION TITLE
Error while opening opening uploaded *.ipynb file

### DISCUSSION THREAD

**Discussion by shailavij** [community/community#21608] [New to GitHub]
<p>Hi  All,</p>
<p>        I am getting error (:Sorry, something went wrong. <a href="https://github.com/shailavij/Hello-World/blob/master/Energy_26_04_2019.ipynb" rel="nofollow noopener">Reload?</a>. ), whenever trying open uploaded file in my repository. Kindly let me know how to resolve the issue </p>
<p>Thanks </p>
<p>     </p>

**Comment by Wabri** [comment, marked as answer]
<p>Hi  <a class="mention" href="/shailavij">@shailavij</a>,</p>
<p>Welcome to the Github Community Forum!</p>
<p>There are no error for me -&gt; <a href="https://github.com/shailavij/Hello-World/blob/master/Energy_26_04_2019.ipynb" rel="nofollow noopener">link</a></p>
<p>-Gabriele-</p>
<p>Mark helpfull posts with Accept as Solution to help other users locate important info. Don’t forget to give Kudos for great contents!</p>

**Comment by docktermj** [comment]
<p>As seen in other issues,   such as:</p>
<ol>
<li><a href="https://github.com/jupyter/notebook/issues/3555" rel="nofollow noopener">https://github.com/jupyter/notebook/issues/3555</a></li>
<li><a href="https://github.com/iurisegtovich/PyTherm-applied-thermodynamics/issues/11" rel="nofollow noopener">https://github.com/iurisegtovich/PyTherm-applied-thermodynamics/issues/11</a></li>
</ol>
<p>This is an intermittent issue that GitHub has not fixed yet.</p>

**Comment by WittmannF** [comment]
<p>&gt; There are no error for me -&gt; <a href="https://github.com/shailavij/Hello-World/blob/master/Energy_26_04_2019.ipynb" rel="nofollow noopener">link</a></p>
<p>This is not a solution. There are many .ipynb files that constantly don’t render. Why does that happen?</p>

**Comment by salihmarangoz** [comment]
<p>I have solved the problem by rendering notebooks by myself and publishing HTML outputs directly. All done automatically using “Github Actions” + “nbconvert” + “Github Pages”. All HTML outputs are pushed to another branch. After a successful rendering previous HTML outputs are being deleted, so not using any extra disk space.</p>
<p>See my repository: <a href="https://github.com/salihmarangoz/github_ci_nbconvert" class="inline-onebox" rel="noopener nofollow ugc">GitHub - salihmarangoz/github_ci_nbconvert: Convert notebook (*.ipynb) files using Github Actions automatically after each commit and access HTML outputs with a public URL using Github Pages</a></p>
